paparazzi-gradle-plugin/src/main/java/app/cash/paparazzi/gradle/PaparazziPlugin.kt

```diff
@@ -138,14 +150,29 @@ public class PaparazziPlugin @Inject constructor(
       val reportOutputDir =
         project.extensions.getByType(ReportingExtension::class.java).baseDirectory.dir("paparazzi/${variant.name}")
 
-      val testInstrumentation = testVariant.instrumentation
-      testInstrumentation.transformClassesWith(
-        ResourcesCompatVisitorFactory::class.java,
-        InstrumentationScope.ALL
-      ) { }
-      testInstrumentation.setAsmFramesComputationMode(
-        FramesComputationMode.COMPUTE_FRAMES_FOR_INSTRUMENTED_METHODS
-      )
+      // TODO: Fixup with for [Android Multiplatform Library Plugin] doesn't have exception with instrumentation transformations for KMP
```

Comment:
Since I couldn't find an issue for thin on the Google tracker I looked into this a bit and opened one https://issuetracker.google.com/issues/462919291. Without deep AGP knowledge it just looks like the plugin is calling 2 methods in the wrong order